app/assets/stylesheets/mobile/mobile.scss

```diff
@@ -647,7 +647,7 @@ form#new_user.new_user input.btn {
   text-shadow: 1px 1px 20px rgb(126, 240, 77);
 }
 
-#conversation_inbox {
+#conversation_inbox, .notifications {
```

Comment:
Is it ok with it? Or should I put it [line 516](https://github.com/SansPseudoFix/diaspora/blob/Fix-pagination-notifications-mobile/app/assets/stylesheets/mobile/mobile.scss#L516)? I do it to avoid duplicate code.